modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -747,6 +748,40 @@ private void notifyLifecycleBeansEx(LifecycleEventType evt) {
         }
     }
 
+    /** */
+    private void ackClassPathElement(File clsPathElement, StringBuilder clsPathContent) {
```

Comment:
Use `org.apache.ignite.internal.util.typedef.internal.SB` instead of `StringBuilder`